test/CMakeLists.txt

```diff
@@ -98,6 +98,7 @@ list( APPEND test_input
 )
 
 list( APPEND test_output
+  testoutput/gdas.t00z.atms_n20_ta_remap.tm00.nc
```

Comment:
@smaticka this is the wrong input file `bufr2ioda.x` converts a BUFR file usually named something like `gdas.t00z.atms.*` so not sure where this input came from